app/main.py

```diff
@@ -1,3 +1,15 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+    cache_item = {}
+
+    def long_time(*args) -> str:
```

Comment:
I think `wrapper` better name 
also `str` wrong annotation, use `Any`
```suggestion
    def wrapper(*args) -> Any:
```